--- origsrc/geoclue-2.1.10/docs/Makefile.am	2014-01-29 12:01:13.000000000 -0600
+++ src/geoclue-2.1.10/docs/Makefile.am	2015-01-06 13:19:36.727994100 -0600
@@ -5,6 +5,9 @@ AUTOMAKE_OPTIONS = 1.7
 # The name of the module.
 DOC_MODULE=geoclue
 
+# Uncomment for versioned docs and specify the version of the module, e.g. '2'.
+DOC_MODULE_VERSION=2
+
 # The top-level SGML file.
 DOC_MAIN_SGML_FILE=geoclue-docs.xml
 
@@ -12,7 +15,7 @@ DOC_MAIN_SGML_FILE=geoclue-docs.xml
 SCAN_OPTIONS=--ignore-headers=config.h --rebuild-sections --rebuild-types
 
 # The directory containing the source code. Relative to $(srcdir)
-DOC_SOURCE_DIR=../src/public-api
+DOC_SOURCE_DIR=$(top_srcdir)/src/public-api
 
 # Used for dependencies
 HFILE_GLOB=
@@ -22,7 +25,7 @@ CFILE_GLOB=
 IGNORE_HFILES= \
 	$(NULL)
 
-GTKDOC_LIBS = $(top_srcdir)/src/public-api/libgeoclue-public-api.la
+GTKDOC_LIBS = $(top_builddir)/src/public-api/libgeoclue-public-api.la $(GTKDOC_DEPS_LIBS)
 
 # Extra options to supply to gtkdoc-mkdb
 MKDB_OPTIONS=--sgml-mode --output-format=xml
